File: desafio-02/Elismar13/python/numeros_primos.py
"""
Autor: Elismar Silva Pereira
Data: 06/10/2020

Desafio 2: listando números primos.

Escreva um programa para listar todos os números primos entre 1 e 10000.
"""

RANGE = 10000

def main():

  def numero_primo(numero: int):
    numDivisoes = 0
    for divisor in range(2, numero):
      if(numero % divisor == 0):
        return False
    return True


  for numero in range(2, RANGE+1):
    primo = numero_primo(numero)
    if(primo):
      print("{} é primo.".format(numero), end="\n")

if(__name__ == "__main__"):
  main()

File: desafio-02/maurilioSo/java/numerosPrimos.java

public class numerosPrimos {

	public static void main(String[] args) {

		/*
		 * Escreva um programa para listar todos os números primos entre 1 e 10000, na
		 * linguagem de sua preferência.
		 */

		int i = 1, qtd =0;

		while (i <= 1000000) {
			int teste = 0;
	
				for (int j = 2; j <= i/2; j++) {

					if (i % j == 0) {
						teste++;
						break;
					}
				}
		
			
			if (teste == 0) {
				System.out.println(i);
				qtd++;

			}
			i++;
			

		}
		System.out.println(qtd);
		

	}

}
